style.css

```diff
@@ -686,14 +686,179 @@ ul {
   left: 0;
 }
 
+/* full card  */
+
+.full-card-cont {
+  width: 80%;
+  margin-left: 10%;
+  backdrop-filter: blur(15px);
+
+  /* display: none; */
+}
+
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly ensure that the pop-up for mobile shows the see live and see source button.

*your design*
![hidden button](https://user-images.githubusercontent.com/59397159/174225239-2c6496b8-b710-410f-931f-912c75f0b238.png)

*figma mobile popup*
![mob](https://user-images.githubusercontent.com/59397159/174225424-e72c2e4a-5b22-4dcf-b48d-36f056509ecd.png)

Reply:
**[REQUIRED CHANGES]**

- Please, kindly ensure your desktop pop header and technologies are well-styled to match the Figma design below:

*figma*
![figmaheader](https://user-images.githubusercontent.com/59397159/174225808-28c37985-3a2c-4046-8aaa-dc0f6df3b546.png)

*yours*
![yourheader](https://user-images.githubusercontent.com/59397159/174225966-c9437d28-01d0-402e-8c47-d11a5ca644d3.png)

Reply:
**[REQUIRED CHANGES]**

**README FILE**

- Please, kindly update your readme to have the current project name.
- Add a getting started header, which tells the users how to have your project locally on their machine.
- Add a second author that has contributed to this project.
- Remove the headers that are not used to make your readme more professional.
- Update the live demo link as what you have is not showing your current project.

Reply:
done